src/Stryker.Core/Stryker.Core/Instrumentation/StaticInitializerMarkerEngine.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Stryker.Core.InjectedHelpers;
+
+namespace Stryker.Core.Instrumentation
+{
+    internal class StaticInitializerMarkerEngine: BaseEngine<ExpressionSyntax>
+    {
+        private const string MutantContextValueTrackName = "TrackValue";
+
+        public StaticInitializerMarkerEngine(string markerId) : base(markerId)
+        {
+        }
+
+        public ExpressionSyntax PlaceValueMarker(ExpressionSyntax node)
```

Comment:
Might be nice to add a xml comment above of how the created code might look like. Reads a lot easier than this syntax factory code

Reply:
done